### ISSUE TITLE
AVT 3 - Carbon X Loading component should be an aria-live region

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1918]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
VoiceOver does not announce the loading status. 

This issue is similar to #1444 and needs further discussion.

**Comment by dakahn** [comment]
Just tested this with NVDA on Windows 10 and it announces the loading status. Then I jumped to my Macbook and yeah, no announcement at all using VoiceOver. 😕

Any ideas as to why that might be?

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.
